test/suite/run.sh

```
@@ -96,38 +96,42 @@ wait_sandbox() {
+  "$HARNESS" apply -o yaml
 
 run_test "parse: custom provider profile" \
-  "$HARNESS" apply --dry-run -f "$CONFIGS/agent-groq.yaml"
+  "$HARNESS" apply -o yaml -f "$CONFIGS/agent-groq.yaml"
 
 run_test "parse: harness with payloads" \
-  "$HARNESS" apply --dry-run -f "$CONFIGS/harness-with-payloads.yaml"
+  "$HARNESS" apply -o yaml -f "$CONFIGS/harness-with-payloads.yaml"
 
 run_test "output: kind: payload in -o yaml" \
   bash -c '"$1" apply -o yaml -f "$2" | grep "kind: payload"' _ "$HARNESS" "$CONFIGS/harness-with-payloads.yaml"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)